src/index.css

```diff
@@ -1,16 +1,9 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+/* Remove focus outline in Firefox */
+@-moz-document url-prefix() {
+  *:focus,
+  *:focus-visible,
+  *:focus-within,
+  *:active {
+    outline: transparent !important;
+  }
 }
```

Comment:
Why these lines are removed?

Reply:
We are importing tailwind.css file in index.res, It already imports the tailwind declarations, which I have removed
<img width="1149" alt="Screenshot 2024-10-23 at 10 53 56 PM" src="https://github.com/user-attachments/assets/ba301a2c-a4ba-4ef8-b14b-f3b7aa259bc3">

There are other css related to body and code tag, which were not used in any files, therefore I removed it